bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -338,6 +348,97 @@ async def nominate_command(self, ctx: Context, user: MemberOrUser, *, reason: st
 
         await self._nominate_user(ctx, user, reason)
 
+    @app_commands.checks.has_any_role(*STAFF_ROLES)
+    async def _nominate_context_callback(self, interaction: discord.Interaction, message: discord.Message) -> None:
```

Comment:
Could be cool if, in the future, we spawn a modal when executing the content menu to capture a reason from the user, which could be pre-filled with their existing resona dn the message jump link.